### DISCUSSION TITLE
LORAWAN working on ESP32S2, RP2040 and SAMD21

### DISCUSSION THREAD

**Discussion by jerryneedell** [jgromes/RadioLib#937] [Show and tell]
Just as a followup to #935  -- after the help received there, I now have working examples (end device) for the following hardware -- using US915:  
Adafruit Feather ESP32s2 with Adafruit RFM95 Featherwing
Adafruit Feather Rp2040 RFM95
Adafruit Feather M0 RFM95
The M0 (SAMD21) does not support "persistent memory" so TTN is not overly happy with it when it resets, but it does work!

This is an exciting new tool!

**Comment by StevenCellist** [comment]
Very happy to hear that you're having successes on all different platforms! Please keep them running & an eye out for weird behaviour, but really glad it's working well :)
